salt/haproxy/haproxy.manifest

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: haproxy
+  namespace: kube-system
+  labels:
+    name: haproxy
+spec:
+  containers:
+  - name: haproxy
+    resources:
+      requests:
+        memory: 128Mi
+      limits:
+        memory: 128Mi
```

Comment:
this will need testing to know if these values are too big or too small - I just made a guess to start with.